picoquic/frames.c

```diff
@@ -1506,25 +1669,27 @@ int picoquic_prepare_connection_close_frame(picoquic_cnx_t* cnx,
             *consumed = 0;
             ret = PICOQUIC_ERROR_FRAME_BUFFER_TOO_SMALL;
         }
-    } else {
+    }
+    else {
         *consumed = 0;
         ret = PICOQUIC_ERROR_FRAME_BUFFER_TOO_SMALL;
     }
 
     return ret;
 }
 
-int picoquic_decode_connection_close_frame(picoquic_cnx_t* cnx, uint8_t* bytes,
-    size_t bytes_max, size_t* consumed)
+int picoquic_decode_generic_close_frame(picoquic_cnx_t* cnx, uint8_t* bytes,
```

Comment:
Same comment as in logging -- the generic thing is short-lived.

(rathole: seems weird that we replicate frame decoding logic twice -- once here and once in logging.)

Reply:
Yes. Should find a way to unify that. And "generic" is not a frame type, just a name for common code.

Reply:
Right.  In draft-13, both frames are identical.  However, in Editor's draft they have different fields already. That was my comment.  Nothing to do here for now, of course, since this is draft-13 support.